### PR TITLE
updated readme.md with dirty codes

### PR DESCRIPTION
**Pull request by FrankYuanEvoco**
modifided README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,3 +26,6 @@ Want to create your presentation using a visual editor? Try the official reveal.
 MIT licensed
 
 Copyright (C) 2011-2020 Hakim El Hattab, https://hakim.se
+
+
+asdjhflqukhrg;ihsdlkjgqb;kherkq
```
Comment (FrankYuanEvoco) [inline README.md]: WTF is this????

**Review by FrankYuanEvoco — COMMENTED** [review]
donot approve it!!!!!!!

File: README.md [hunk 2]
```diff
@@ -26,3 +26,6 @@ Want to create your presentation using a visual editor? Try the official reveal.
 MIT licensed
 
 Copyright (C) 2011-2020 Hakim El Hattab, https://hakim.se
+
+
```
Comment (pezlewei) [inline README.md:30]: stupid!!!

**Review by pezlewei — COMMENTED** [review]
i dont know!!!

File: README.md (hunk 2 above)
Comment (FrankYuanEvoco) [reply README.md:30]: check latest change

**Pull request merged**